standard/expressions.md

```diff
@@ -968,6 +968,16 @@ For a function member that includes a parameter array, if the function member is
   - for a fixed value parameter or a value parameter created by the expansion, an implicit conversion ([§10.2](conversions.md#102-implicit-conversions)) exists from the argument expression to the type of the corresponding parameter, or
   - for a `ref` or `out` parameter, the type of the argument expression is identical to the type of the corresponding parameter.
 
+In the case of a method group
+
+- If the group contains both instance and static members, instance members invoked without an instance receiver or context are not applicable.
+- If the group contains both instance and static members, static members invoked with an instance receiver are not applicable.
+- When there is no receiver, static members in a static context are applicable; otherwise, both static and instance members are applicable.
+- When the receiver is ambiguously an instance or type due to a color-color situation, both are applicable.
```

Comment:
"due to a color-color situation" doesn't sound like something that should be in the normative text. (Perhaps in an example, or refer to an existing example?)

Reply:
Came to to make a similar comment and found Jon had beaten me to it. A Google of the phrase only seems to turn up references to the use here...

Reply:
Yup, there's still work to do on this bit :)

Reply:
Go with suitable wording around @gafter's reference to §11.7.6.2. Not sure I agree that _ambiguous_ is perhaps not the correct word, but if it is changed here it also needs to be changed in §11.7.6.2

Reply:
11.7.6.2 specifically says that it is *not* ambiguous.

Maybe something like

> When the receiver could be either an instance or a type as described in §11.7.6.2, both instance and static methods are applicable.

Reply:
Done.